### ISSUE TITLE
PhoneMetadataCollection goog.require'd but not used

### ISSUE THREAD

**Issue by 4u** [googlei18n/libphonenumber#969]
I use closure compiler with `--jscomp_error=extraRequire` flag. Could you please remove goog.require calls with unused modules? Thank you.

```
/path/to/libphonenumber/javascript/i18n/phonenumbers/asyoutypeformatter.js:38: ERROR - 'i18n.phonenumbers.PhoneMetadataCollection' goog.require'd but not used
goog.require('i18n.phonenumbers.PhoneMetadataCollection');
^

/path/to/libphonenumber/javascript/i18n/phonenumbers/asyoutypeformatter.js:39: ERROR - 'i18n.phonenumbers.PhoneNumber' goog.require'd but not used
goog.require('i18n.phonenumbers.PhoneNumber');
^

/path/to/libphonenumber/javascript/i18n/phonenumbers/asyoutypeformatter.js:40: ERROR - 'i18n.phonenumbers.PhoneNumber.CountryCodeSource' goog.require'd but not used
goog.require('i18n.phonenumbers.PhoneNumber.CountryCodeSource');
^

/path/to/libphonenumber/javascript/i18n/phonenumbers/asyoutypeformatter.js:41: ERROR - 'i18n.phonenumbers.PhoneNumberDesc' goog.require'd but not used
goog.require('i18n.phonenumbers.PhoneNumberDesc');
^

/path/to/libphonenumber/javascript/i18n/phonenumbers/asyoutypeformatter.js:43: ERROR - 'i18n.phonenumbers.metadata' goog.require'd but not used
goog.require('i18n.phonenumbers.metadata');
^

/path/to/libphonenumber/javascript/i18n/phonenumbers/phonenumberutil.js:45: ERROR - 'i18n.phonenumbers.PhoneMetadataCollection' goog.require'd but not used
goog.require('i18n.phonenumbers.PhoneMetadataCollection');
^

6 error(s), 0 warning(s)
```

**Comment by 4u** [comment]
https://github.com/4u/libphonenumber/commit/73412c66a2bcd8f8c0d9a6f0a20f0129ed25c5fd

**Comment by penmetsaa** [comment]
@4u
This looks like a bug. We'll fix it in an upcoming release. 
Thanks for taking the time to improve libphonenumber!

**Comment by penmetsaa** [comment]
This is fixed as part of PR #1671
